UI development

TristanFish opened the pull request.

Takagi1 committed:
Merging Third person Character and edit, merged health bar variables.

elvimoon committed:
Edited to allow steam connection

elvimoon committed:
Updated so friend list/username shows up

elvimoon committed:
Added ServerList widget, WIP

keshy101 reviewed (approved):
weekly merge before issues with huge merge

elvimoon committed:
Connected player hud to third person character for testing health bar

elvimoon committed:
small adjustments for pre-packaging demo

third person character:
1. max health default value from 0 to 100.
2. created widget on begin player to add player hud widget to viewport

Takagi1 committed:
Added functionallity to ServerList

Takagi1 committed:
Revert "Added functionallity to ServerList"

This reverts commit 2de53db68b4a62934eec6fc9d37f76580a6d0f76.

Takagi1 committed:
Revert "Revert "Added functionallity to ServerList""

This reverts commit 8be7860b18d0bd1166191d136e8f3904d14ceb1f.

Takagi1 committed:
Merge branch 'Ui-Development' of https://github.com/Humber-Game-Production/GAME259-SectionB into Ui-Development

ConorLong reviewed (approved):
Big nice.
